src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
It is arbitrary.  But the HttpClient.Timeout value has to be greater than the expected long duration of the test server (40 seconds in this case).

If you are increasing the timeout value on HttpClient, then I assume it is because you feel that the  test server (3rd party server) is really taking even longer than the 40 seconds we asked it to delay.

The point of this test is to verify that the operation of HttpClient Timeout feature is working as expected.  And we choose 60 seconds because it is the non-default value of that property.

It sounds like you still want to keep the test semantically the same....i.e. test that having an HttpClient.Timeout value > (the actual delay time of the server) means that the request will succeed.  In that case, you can try setting it higher.  But it is very unusual to need 5 minutes, for example, to get a response back from the server.  If you are waiting that long, then there is something else wrong with the tests/infrastructure.  And, having a long timeout like that means we will never want this test to be InnerLoop because it will make inner loop tests slow.